spec/integration/hanami/router/params_spec.rb

```diff
@@ -71,6 +74,28 @@
     end
   end
 
+  context "file upload" do
+    subject do
+      r = router
+
+      Rack::Builder.new do
+        use Hanami::Middleware::BodyParser, :form
+        run r
+      end
+    end
+
+    it "handles file upload" do
```

Comment:
@jodosha I tried to fix the spec and tried to use `Rack::Multipart.build_multipart` instead of `multipart_fixture`. It seems to work:
```ruby
    it "handles file upload" do
      file = Rack::Multipart::UploadedFile.new(fixture_path("foo.xml"))
      data = Rack::Multipart.build_multipart("file" => file)
      env = Rack::MockRequest.env_for(
        "/submit",
        "CONTENT_TYPE" => "multipart/form-data; boundary=AaB03x",
        "CONTENT_LENGTH" => data.length.to_s,
        method: "POST",
        :input => StringIO.new(data)
      )

      subject.call(env)

      expect(env["router.params"][:file][:filename]).to eq("foo.xml")
      expect(env["router.params"][:file][:tempfile].read).to eq(File.new(fixture_path("foo.xml")).read)
    end
```

Reply:
@wuarmin Thank you 👍